src/PeriodEnd/PaymentsDue/DeployPaymentsDue/sql/MigrationScripts/001_DEDS.dbo.DasAccounts_Add_TransferAllowance.sql

```diff
@@ -0,0 +1,15 @@
+﻿ 
+IF NOT EXISTS (
+SELECT null FROM sys.columns c 
+	INNER JOIN sys.tables t 
+		ON c.object_id = t.object_id
+	INNER JOIN sys.schemas s
+		ON t.schema_id = s.schema_id
+	WHERE t.name='DasAccounts'
+	AND s.name='dbo'
+	AND c.name='TransferAllowance'
+)
+BEGIN
+ALTER TABLE  dbo.DasAccounts
+	Add TransferAllowance decimal(15, 2) NULL
+END
```

Comment:
Is there a risk this might run in a scenario where the table doesn't exists?

Reply:
also why 15,2?

Reply:
I've done a check against sys.tables, sys.columns and sys.schemas so that should guarantee existence or not.

it's 15,2 as the other field Balance is the same. I can double check with Robin on that just in case they produce it differently (we just get decimals over the wire from the accounts api)

Reply:
ah ok, I've seen some of the amount fields as 15,5 which is why I asked. But if that's correct then great